text/src/autogluon/text/text_prediction/predictor/predictor.py

```diff
@@ -252,14 +256,38 @@ def fit(self,
         :class:`TextPredictor` object. Returns self.
         """
         assert self._fit_called is False
+        is_continue_training = self._model is not None
         verbosity = self.verbosity
         if verbosity is None:
             verbosity = 3
-        if presets is not None:
-            preset_hparams = ag_text_presets.create(presets)
+        if save_path is not None:
+            self._path = setup_outputdir(save_path, warn_if_exist=True)
+        if is_continue_training:
```

Comment:
Maybe add TODO: enforce version check. If predictor is from 0.3 and AG is installed from 0.4, this probably isn't a good idea -> may crash or have strange errors.

Reply:
It is currently backward compatible. Basically, we can load a predictor trained via AutoGluon 0.1--0.3 and continue train the model. The new model will be saved in the latest AutoGluon format.